### DISCUSSION TITLE
Search box in Fly Pie

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [Schneegans/Fly-Pie#346] [Fly-Pie Q&A]
Hello, 
Is there any possibility to have a search box and search for a term in specific website (not Google) ?
Thanks

**Comment by Schneegans** [comment]
Hi there! I think you have to describe your request in much more detail.
* What do you mean with "search box"? Should there be the possibility to somehow insert text somewhere after an item was selected in Fly-Pie? Or directly _into_ a menu item? How should this look like? Could you provide a drawing?
* What do you mean by "search for a term"? Like in a search engine? Or rather like Ctrl+F searching in some website?

Even if I do not know what you want, I am pretty sure that this is not possible right know. You could open a proper feature request in the [issues tracker](https://github.com/Schneegans/Fly-Pie/issues) providing all these details.

However, please keep in mind that from my side, Fly-Pie is currently in low-maintenance mode as I am investing most of the little spare time I have into its successor project, [Kando](https://github.com/kando-menu/kando). So I personally will not implement new features for Fly-Pie these days. It's more likely that I add something like this into Kando.

**Reply by Shahin-rmz** [reply]
Excited to see the new Launcher.
The feature request has been [opened](https://github.com/Schneegans/Fly-Pie/issues/347).
Thanks
